src/ui/Controllers/CourseController.cs

```diff
@@ -89,5 +100,12 @@ public CourseController(ManageApi manageApi)
 
             return View(viewModel);
         }
+
+        private void Validate(string instCode, string accreditingProviderId, string ucasCode)
+        {
+            if (string.IsNullOrEmpty(instCode)) throw new ArgumentNullException(instCode, "instCode cannot be null or empty");
```

Comment:
Add braces.

https://nakedsecurity.sophos.com/2014/02/24/anatomy-of-a-goto-fail-apples-ssl-bug-explained-plus-an-unofficial-patch/

Reject.

Reply:
All done